tests/test_imports.py

```diff
@@ -0,0 +1,54 @@
+import os
+import imp
+import pkgutil
+import sys
+import unittest
+
+
+class ImportTestSuite(unittest.TestCase):
+
+    def test_import_of_each_module(self):
+        """Import each module of package git_cc.
+
+        This test was added because the import Python 3 requires you to use the
+        relative package import syntax "from . import <module name>" to import
+        packages from the same package, whereas Python 2 you could just use
+        "import <module name>".
```

Comment:
It seems unfortunate there's no easier way to verify the source is compatible with a python version? :(